src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker_Patterns.cs

```diff
@@ -79,7 +79,6 @@ public override BoundNode VisitITuplePattern(BoundITuplePattern node)
         /// </summary>
         /// <param name="inputType">Type type of the input expression (before nullable analysis).
         /// Used to determine which types can contain null.</param>
-        /// <returns>true if there is a top-level explicit null check</returns>
         private void LearnFromAnyNullPatterns(
```

Comment:
>LearnFromAnyNullPatterns [](start = 21, length = 24)

This will have to be modified to look for bound patterns that are explicit non-null tests.

Reply:
Added.

---
In reply to: [365461994](https://github.com/dotnet/roslyn/pull/40892#discussion_r365461994) [](ancestors = 365461994)